Locales/Russian_ru.json

```diff
@@ -111,7 +111,7 @@
 	"Pick tunnel to travel":"Выберите тоннель для перемещения",
 	"Return to base":"Вернуться на базу", //OLD
 	"Travel right":"Переместиться вправо",
-	"Travel left":"Переместиться влево",
+	"Travel ":"Переместиться влево",
```

Comment:
Why was "left" removed? The translation appears to still be referring to travelling left, and other languages have this entry as "Travel left".